packages/devextreme/js/__internal/ui/html_editor/modules/m_tableContextMenu.ts

```diff
@@ -71,6 +71,15 @@ if (Quill) {
       eventsEngine.off(this.editorInstance._getContent(), CONTEXT_MENU_EVENT);
     }
 
+    _escKeyHandler() {
+      this._contextMenu.hide();
+      this.editorInstance.focus();
+    }
+
+    _addEscapeHandler(e) {
```

Comment:
Let's name it a bit wider while this method if passed to initialized
```suggestion
    _onContextMenuInitialized(e) {
```

Reply:
[d538607](https://github.com/DevExpress/DevExtreme/pull/28871/commits/d53860751c583718c36f482aa1dfd65cc01888f9)